packages/vite/src/node/ssr/fetchModule.ts

```diff
@@ -137,6 +137,7 @@ export async function fetchModule(
   return {
     code: result.code,
     file: mod.file,
+    serverId: mod.id!,
```

Comment:
This is awesome! So good to see Astro working 🎉
Could this be called `id` directly? Is the id in the runner different than the one in the server?

Reply:
Yes, they are different. Not intentionally, really. An artifact from vite-node that I am too afraid to change 😄 

The ID in the module runner is the URL on the server module graph. The module runner caches by urls so it's easier to keep track of, it doesn't have multiple cache maps

Reply:
Ok, let's merge this. Ok, if this is part of the public API for ModuleRunner. I'm thinking that we could maybe use `url` then everywhere we have `id` in the runner? 🤔
And then `serverId` can be just `id`.

Reply:
Yeah, I think so